bedrock/base/templates/includes/sticky-promo.html

```diff
@@ -0,0 +1,38 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+<aside class="mzp-c-sticky-promo mzp-t-product-firefox mzp-t-dark">
+  <button class="mzp-c-sticky-promo-close" type="button">{{ ftl('ui-close') }}</button>
+  <div class="promo-firefox">
+    <h3 class="mzp-c-sticky-promo-title">{{ ftl('firefox-sticky-promo-get-the-latest-firefox') }}</h3>
+    {{ download_firefox_thanks(dom_id='sticky-promo', download_location='other', button_class='mzp-c-button mzp-t-product mzp-t-small') }}
+  </div>
+  <div class="promo-products">
+    <h3 class="mzp-c-sticky-promo-title">{{ ftl('firefox-sticky-promo-meet-our-family-of') }}</h3>
+    <ul class="promo-products-list">
+      <li>
+        <a data-link-name="Browsers" data-link-type="link" class="promo-products-link" href="{{ url('firefox.browsers.index') }}">
+          <img src="{{ static('protocol/img/logos/firefox/browser/logo.svg') }}" alt="">
+          Browsers
+        </a>
+        <a data-link-name="Monitor" data-link-type="link" class="promo-products-link" href="https://monitor.firefox.com/{{ referral|safe }}" rel="external noopener">
```

Comment:
Same question for external links: Are there more tags needed? Do we need to label all of these links with a `sticky-promo` sort of GTM ID?

Reply:
The issue here is that this file is included into pages that already have `referrals` defined. Is it okay to overwrite them here? 

Example: `bedrock/firefox/templates/firefox/home/index-master.html`
L48 sets the `referrals`, L60 includes the `sticky-promo.html` file.